manuscript/manuscript.Rmd

```diff
@@ -491,68 +491,51 @@ Many of our most realistic ecological models exist only as numerical simulations
 
 <!-- Solving 'real' problems will be hard -->
 Successfully applying RL to complex ecological problems is no easy task.
-The flexibility of the approach makes it relatively straightforward to include realistic biology and environmental science in framing an RL problem, but training an RL agent to reach an acceptable level of performance is another task.
+The flexibility of the approach makes it relatively straightforward to include realistic biology and environmental science in framing an RL problem, but training an RL agent to reach an acceptable level of performance can often be more challenging than expected.
 As the examples we have presented illustrate, it is important to begin with simple problems, including those for which an optimal strategy is already known.
 Such examples provide important benchmarks against which to calibrate the performance, tuning and training requirements of RL,
 and can easily be extended into more complex problems once RL agents can master the basics.
 Finding acceptable policies even in relatively simple problems may prove beyond the reach of existing algorithms and available computing power.
-<!-- Marcus is working on tightening up these next few paragraphs, precisely which issues actually get highlighted TBD-->
-In seeking either to adopt or assess a deep RL-based approach to an ecological issue, we highlight three key issues that are important to bear in mind: instability, interpretability, and reward design.
```

Comment:
Seems like we still need to acknowledge instability as a fundamental issue?  I think the explicit observation that even the random seed impacts the outcomes in far more significant ways than it does in the many classical optimization algorithms that also use pseudo-random number, is an important concrete thing to mention that is kinda lost in the broader discussion of generalizability? 

I think the previous paragraph on instability wasn't perfect but identified concrete issues and had some concrete recommendations, the new paragraph on skills feels a lot more vague by comparison.

Reply:
Yeah, I took it out because I think of building more general agents as the bigger issue. Instability is a practical pain but if you find a good set of hyperparameters and an initialization seed then this problem is over. Building an agent that can generalize past experiences to new situations is a problem that has no solid answers to right now. I also think of generalization as a more likely failure mode than instability in terms of getting RL agents to inform decisions in the real world. Like if the agent is unstable in training, the consequence is that you will keep training until you get a good working agent. But if the agent fails to generalize to the real world, the consequence is that you could make horrible decisions because of this.